client/src/app/site/motions/models/view-category.ts

```diff
@@ -64,6 +64,14 @@ export class ViewCategory extends BaseViewModel<Category> implements CategoryTit
         return name;
     }
 
+    public get nameWithParentAbove(): string {
```

Comment:
Please think of categories with multiple parents in the category-tree. Either document, that only the direct parent is used, or fix this function, that every parent is used (I do not know, if this is intended or not..)

Reply:
According to my information, thouse are irrelevant.